src/firefly/js/metaConvert/vo/ObsCoreConverter.js

```diff
@@ -194,7 +194,7 @@ async function getObsCoreSingleDataProduct({table, row, activateParams, serviceD
         const positionWP= getSearchTarget(table.request,table) ?? makeWorldPtUsingCenterColumns(table,row);
         const request= makeObsCoreRequest(dataSource, positionWP, titleStr,table,row);
         const primDPType= doFileAnalysis ?
-            await uploadAndAnalyze({request,table,row,activateParams,serviceDescMenuList}) :
+            await uploadAndAnalyze({request,table,row,activateParams,serviceDescMenuList,originalTitle:request.getTitle()}) :
```

Comment:
Will there ever be a case where ```originalTitle``` is not ```request.getTitle()```?  If not, then maybe no need to add another prop to an already long list of props.

Reply:
no it is computed